### ISSUE TITLE
Jquery bootgrid with bootstrap 4

### ISSUE THREAD

**Issue by DalamarX** [rstaib/jquery-bootgrid#397]
Using bootstrap 3.3.7 I think it was, the bootgrid looks and works properly.  Moving up to the latest bootstrap 4.1.1, and everything looks off.  Am I missing something obvious?

**Comment by ArneRiseth** [comment]
Not sure if this is of any help but..

There are changes in Bootstrap 4, like 
- Glyphicons no longer baked into it,I used font awesome instead.
- Think pull-right is changed in bootstrap 4 or removed and I used dropdown-menu-right instead.
- btn-default was renamed to btn-secondary (think i ended up using something different)
- input-group-addon was removed so i think i used input-group-prepend instead.
- table-condensed is renamed to table-sm

More general info on migrating to bootstrap 4 here :
https://getbootstrap.com/docs/4.0/migration/
